VSCode/armsnippets.json

```diff
@@ -1489,4 +1482,52 @@
         "   }"
     ],
     "description": "Azure Kubernetes Service Cluster"
-}
+},
+"Azure Logic App": {
+    "prefix": "arm-logicapp",
+    "body": [
+      "{",
+      "  \"type\": \"Microsoft.Logic/workflows\",",
+      "  \"name\": \"${1:logic-app-name}\",",
+      "  \"apiVersion\": \"2017-07-01\",",
+      "  \"location\": \"[resourceGroup().location]\",",
+      "  \"tags\": {},",
+      "  \"properties\": {",
+      "      \"state\": \"Disabled\",",
```

Comment:
Why disabled?

Reply:
Because logic apps are event driven, which means that they fire off as soon as the trigger is satisfied. Therefore better to deploy them as disabled first, just to be safe